src/components/diagrams/singleLineDiagram/single-line-diagram.js

```diff
@@ -756,7 +780,7 @@ const SingleLineDiagram = forwardRef((props, ref) => {
                     <AlertInvalidNode noMargin={true} />
                 </Box>
             ) : (
-                <Box position="relative">
```

Comment:
Why removing ? It's useless ?

Reply:
That change is what enable the full screen buttons to be aligned. Previously its position was relative to the svg container, by removing this line it is now relative to sld pane

Reply:
Ah yes, I see